client/src/Pages/Login.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from "react";
+import Navbar from "../Components/Navbar";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import i18 from 'i18next'
+import { Link } from "react-router-dom";
+
+const Login = () =>{
+
+    const [values,setValues] = useState({
+        gmail: '',
+        password: ''
+    })
+
+    const [isloggedIn, setisloggedIn] = useState(false);
+
+    useEffect(() => {
+        axios.get('http://localhost:3030/auth/status', { withCredentials: true })
+            .then(response => {
+                console.log('Auth status:', response.data);
+                setisloggedIn(response.data.loggedIn);
+            })
+            .catch(err => {
+                console.error('Error checking login status:', err);
+            });
+    }, [isloggedIn]);
+    
+    
+
+    const navigate = useNavigate();
+
+    const getLanguagePath = () => `/${i18.language}`
+
+    axios.defaults.withCredentials = true;
+
+    const handleSumbit = (e) =>
+        {
+            e.preventDefault();
+            if(isloggedIn)
+            {
+                alert('You are already logged in');
+                return;
+            }
+            if(values.gmail==='' || values.password==='')
+            {
+                alert('Make sure to fill both fields.')
+                return;
+            }
+            axios.post('http://localhost:3030/auth/login', values)
+            .then(res => 
+                {
+                    console.log('Server response:', res.data); // Log the response data
+                    if(res.data.Status === 'Success') 
+                    {
+                        navigate(`${getLanguagePath()}/`);
+                    }
+                    else if(res.data.Error==='No email existed')
+                    {
+                        alert('Incorrect Email :/')
+                    }
+                    else if(res.data.Error==='Password not matched')
+                    {
+                        alert('Incorrect Password :/')
+                    }
+                    else 
+                    {
+                        alert('Error while registering the user.');
+                    }
```

Comment:
This looks like the case when user was not **authenticated** so the error log is misleading. Also where is the handling for registering a first time user?

Reply:
This error is for that if the user email and password, isn't find in database so this error will be pop-out.
But, for now it is not important as we are using auth0 as user management system.